tests/classnames.macro.test.js

```diff
@@ -11,6 +11,22 @@ function run(code) {
   }).code.trim()
 }
 
+test("variable", (t) => {
+  const input = stripIndent`
+    import classNames from '../src/classnames.macro';
+    const something = false;
+    const CLASS_NAMES = classNames(something, "py-2");
+  `
+
+  const expected = stripIndent`
+    const something = false;
+    const CLASS_NAMES = "py-2" + (" " + (something || ""));
```

Comment:
I had this use case in mind but did not look into resolving variables values in Babel. I know it’s possible to produce following output in such case:

```js
const CLASS_NAMES = "py-2"
```

Do you know how to do it? Should we?

Reply:
Will change the commit message :)

I dont think it matters since terser will take care of it, test the code on https://xem.github.io/terser-online/

`const something = false;
const CLASS_NAMES = "py-2" + (" " + (something || ""));
console.log(CLASS_NAMES)`

Will produce the following output:

`console.log("py-2 ")`

Reply:
You’re right. Good catch.

Reply:
Commit message changed